### PR TITLE
Update full website

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Math Magician Full Web**
In this project Math Magician built in React to do simple calculation applications like multiplication division subtraction and addition operations.
It has one interface with three parts (numerical party, operator party, and display part).
The project consists of 3 pages: Home, Calculator, and Quote.
build with: 
- React
- linter
- babelrc
- webpack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 
-It have one interface with three parts (numerical party, operator party and display part)
-> ![screenshoot](./Capture.PNG)
+It have one interface with three parts (numerical party, operator party and display part).
+
+> The project consisting of 3 pages: Home, Calculator, and Quote.
+
+> ![screenshoot](./Appscreenshoot.PNG)
+
+# Getting Started with Create React App
```
Comment (Lameck1) [inline README.md:27]: - [ ] At this stage, the README should be informing us on how to get started with this particular App and not CRA. Generally, the README needs editing to match [this particular template](https://github.com/microverseinc/readme-template). Please ensure the README is updated to help make this repo look professional.

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,24 @@
-import react from 'react';
-// eslint-disable-next-line import/extensions
-import Calculator from './components/calculator';
+import { Switch, Route } from 'react-router-dom';
+import Calculator from './Components/Calculator';
+import Header from './Components/header';
+import Quote from './Components/quotePage';
+import Welcome from './Components/welcome';
```
Comment (Lameck1) [inline src/App.js]: - [ ] All these imports do not work. The App fails to compile and the error below is presented:  
  image
  
  WHY?
  The directory with the name `Components` should actually be `components` with lowercase `c`.
  Please refactor accordingly.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Mucyosoda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @Mucyosoda 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

-  n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**